src/lib.rs

```diff
@@ -37,6 +40,12 @@ use pgx::{pg_getarg, prelude::*};
 pub use tests::pg_test;
 pgx::pg_module_magic!();
 
+/// this function will raise an ERROR if the `plrust` language isn't installed
+pub(crate) fn plrust_lang_oid() -> Option<pg_sys::Oid> {
+    static PLRUST_LANG_NAME: &[u8] = b"plrust\0"; // want this to look like a c string
+    Some(unsafe { pg_sys::get_language_oid(PLRUST_LANG_NAME.as_ptr().cast(), false) })
```

Comment:
Hm. Why do we return `Option` here, in light of this? Just to make forward compatibility easier?
```suggestion
    // SAFETY: We pass `missing_ok: false`, which will cause an error if not found.
    // So we always will return a valid Oid if we return at all.
    // The first parameter has the same requirements as `&CStr`.
    Some(unsafe { pg_sys::get_language_oid(PLRUST_LANG_NAME.as_ptr().cast(), false) })
```